### DISCUSSION TITLE
Is there a way to transfer data from one plausable server to another?

### DISCUSSION THREAD

**Discussion by hmzisb** [plausible/analytics#1667] [Self-Hosted Support] [answered]
I am looking to move away from Digital Ocean to AWS. I wanted to know if there is a way to transfer the server without losing the data. Thanks for the help.

**Comment by ukutaht** [comment, marked as answer]
It's a manual job at the moment. You have to know how to work with Clickhouse and Postgres to get exports from both and to import them into a new DB. This is an obvious candidate to automate in the future, but for now we don't have anything more convenient, sorry.

**Reply by Vigasaurus** [reply]
I've had good success with backing up and restoring the entire Docker volumes across instances - it works pretty great, assuming you maintain the same configurations etc. It's also much easier than trying to transfer actual Clickhouse and Postgres backups.

**Reply by Coroner** [reply]
Can you write something more?

**Reply by Vigasaurus** [reply]
https://github.com/plausible/analytics/discussions/1459#discussioncomment-1628791

**Reply by julianengel** [reply]
Dropping the link here as well, it is a manual process but can be done: https://dev.to/codeero/migrating-plausible-data-between-instances-30ng

**Comment by julianengel** [comment]
We had an issue this weekend where our docker manager (caprover) got corrupted and we were forced to migrate to a new server. 
It took a lot of hard work and issues, headaches and challenges, but we managed to get it done. 

I threw everything into a quick post if anyone else has the same issue, you can find it here dev.to [Migrating Plausible Data between Instances/Servers](https://dev.to/codeero/migrating-plausible-data-between-instances-30ng) 

Thank you for an amazing product @ukutaht & @metmarkosaric!
